### ISSUE TITLE
The "move" command doesn't work anymore (macOS Monterey 12.6.3)

### ISSUE THREAD

**Issue by alin89c** [BlueM/cliclick#168]
First I got this warning in Terminal:
_WARNING: Accessibility privileges not enabled. Many actions may fail.
         Enable the checkbox for the containing app in:
         System Preferences → Security & Privacy → Accessibility_

Then, I ran "cliclick m:0,0" (got no warning, but the mouse pointer still didn't move).
The "click" command works though...

Is this a known issue?

**Comment by joelc0193** [comment]
Looks like it works if you move the cursor **twice**:

`cliclick m:0,0 m:100,100`

**Comment by alin89c** [comment]
Hi, and thanks! Tried it and it works (on my OpenCore Ventura 13.3.1 Retina Mid 2012 MacBook Pro). Good job!

**Comment by joelc0193** [comment]
> Hi, and thanks! Tried it and it works (on my OpenCore Ventura 13.3.1 Retina Mid 2012 MacBook Pro). Good job!

is was ChatGPT who gave me the idea while debugging

**Comment by BlueM** [comment]
As I haven’t been able to reproduce this behavior (neither #169 on a different Mac running macOS 13), I don’t see how I could provide a fix.

**Comment by shuiwujiao** [comment]
ChatGPT suggests that the terminal application itself may not be authorized to use "accessibility". For example, if you connect through a terminal or sshd, you need to allow the terminal or sshd to use "accessibility".
This method works well on my PC.

**Comment by alin89c** [comment]
Maybe this issue is related only to Macs running an officially unsupported MacOS... Good thing I have this workaround (to input the values twice).

**Comment by jamesdh** [comment]
I just now tried cliclick myself for the first time and experienced this issue. @joelc0193's [comment](https://github.com/BlueM/cliclick/issues/168#issuecomment-1575103405) worked for me also.

**Comment by jamesdh** [comment]
@BlueM for what it's worth, I only experience this bug using iTerm. It doesn't reproduce using Terminal.
